theano/tensor/tests/test_basic.py

```diff
@@ -1937,14 +1965,89 @@ def test_py_c_match():
 J1Tester = makeBroadcastTester(
     op=tensor.j1,
     expected=expected_j1,
-    good=_good_broadcast_unary_j,
+    good=_good_broadcast_unary_bessel,
+    grad=_grad_broadcast_unary_bessel,
     eps=2e-10,
     mode=mode_no_scipy,
     skip=skip_scipy)
+
 J1InplaceTester = makeBroadcastTester(
     op=inplace.j1_inplace,
     expected=expected_j1,
-    good=_good_broadcast_unary_j,
+    good=_good_broadcast_unary_bessel,
+    grad=_grad_broadcast_unary_bessel,
+    eps=2e-10,
+    mode=mode_no_scipy,
+    inplace=True,
+    skip=skip_scipy)
+
+JvTester = makeBroadcastTester(
+    op=tensor.jv,
+    expected=expected_jv,
+    good=_good_broadcast_binary_bessel,
```

Comment:
I have added `test_ verify_jv_grad` and `test_verify_iv_grad` functions to `test_basic.py` which wrap the ops to fix the first inputs as described in the documentation. I could not figure out how to do this correctly for the inplace versions though - when running `nosetests` on equivalent functions for the inplace ops, either an exception was raised that the (fixed) first input was non-destructible as a constant, or when tried to workaround this by making the passed first input a shared variable rather than constant, the grad numerical results have a large error. Is there a correct way to verify gradients of inplace ops with a subset of inputs fixed?